functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -115,6 +115,12 @@ def get_pods_and_statuses(db_cluster: ScyllaPodCluster, namespace: str, label: s
             yaml.safe_load(pods.stdout)["items"] if pod]
 
 
+def get_replicaset_name_and_owner_kind(db_cluster: ScyllaPodCluster, namespace: str, replicaset_name: str):
```

Comment:
After the addressing of the below comments this one becomes unused and can be deleted.